packages/app-store/zohocalendar/_metadata.ts

```diff
@@ -0,0 +1,27 @@
+import type { AppMeta } from "@calcom/types/App";
+
+import _package from "./package.json";
+
+export const metadata = {
+  name: "Zoho Calendar",
+  description: _package.description,
+  installed: true,
+  type: "zoho_calendar",
+  title: "Zoho Calendar",
+  imageSrc: "/api/app-store/zohocalendar/icon.svg",
+  variant: "calendar",
+  category: "calendar",
+  categories: ["calendar"],
+  logo: "/api/app-store/zohocalendar/icon.svg",
```

Comment:
```suggestion
  logo: "icon.svg",
```

Using relative URL keeps it simple